docs/admin/authorization.md

```diff
@@ -67,6 +67,8 @@ A request has 5 attributes that can be considered for authorization:
         resource is the empty string.
```

Comment:
change to "the resource is the empty string, and the non resource path is populated instead"